Viperinius.Plugin.SpotifyImport/Configuration/playlistConfig.js

```diff
@@ -293,7 +293,7 @@ export default function (view) {
 
             users.forEach(user => {
                 // match a given spotify id with or without a prepended url or uri part, ignore url params
-                const match = /^(https?:\/\/open\.spotify\.com\/user\/|spotify:user:)?([a-zA-Z0-9]+)(\?si=.*)?$/gm.exec(user.Id);
+                const match = /^(https?:\/\/open\.spotify\.com\/user\/|spotify:user:)?(.+)(\?si=.*)?$/gm.exec(user.Id);
```

Comment:
```suggestion
                const match = /^(https?:\/\/open\.spotify\.com\/user\/|spotify:user:)?([^/?\s]+)(\?si=.*)?$/gm.exec(user.Id);
```

Reply:
yeah, I did not consider those cases. Your regex looks better.